openstreetmap-carto.style

```diff
@@ -11,6 +11,7 @@ node,way   addr:housename      text  linear
 node,way   addr:housenumber    text  linear
 way        addr:interpolation  text  linear
 node,way   admin_level  text         linear
+node,way   advertising  text         polygon
```

Comment:
I think this is not a good idea - it would mean database migration. We add keys from hstore like tags->'intermittent'.